src/tests/unit/tests/electron/platform/android/mock-adb.test.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { cloneDeep } from 'lodash';
+import * as path from 'path';
+import {
+    commonAdbConfigs,
+    MockAdbConfig,
+    simulateNoDevicesConnected,
+    simulatePortForwardingError,
+    simulateServiceInstallationError,
+    simulateServiceLacksPermissions,
+    simulateServiceNotInstalled,
+} from 'tests/miscellaneous/mock-adb/setup-mock-adb';
+
+describe('mock-adb tests match snapshots after normalizing path', () => {
+    const definedConfigs = ['single-device', 'multiple-devices', 'slow-single-device'];
```

Comment:
```suggestion
    const definedConfigs = Object.getOwnPropertyNames(commonAdbConfigs);
```

Reply:
I'll put this into a separate PR since I have work staged that needs this change